portal-web/test/functional/com/liferay/portalweb/macros/MySecondMacro.macro

```diff
@@ -0,0 +1,27 @@
+<definition>
+	<command name="navigateToDocuments">
+		<execute function="Click" locator1="LRSelectDocumentsPortlet#OPEN_NAVIGATION_BAR" />
+		<execute function="Click" locator1="LRSelectDocumentsPortlet#CONTENT_DROP_FIELD" />
+		<execute function="Click" locator1="LRSelectDocumentsPortlet#DOCUMENTS_MEDIA_PORTLET" />
+	</command>
+
+	<command name="navigateToBlogs">
+		<execute function="Click" locator1="LRSelectDocumentsPortlet#OPEN_NAVIGATION_BAR" />
+		<execute function="Click" locator1="LRSelectDocumentsPortlet#CONTENT_DROP_FIELD" />
+		<execute function="Click" locator1="LRAddBlog#BLOGS_PORTLET" />
+	</command>
+
+	<command name="addNewDocument">
+		<execute function="Click" locator1="LRSelectDocumentsPortlet#ADD_FILE_BUTTON" />
+		<execute function="Click" locator1="LRSelectDocumentsPortlet#ADD_BASIC_DOCUMENT" />
+	</command>
+
+	<command name="addBlogTag">
+		<execute function="Click" locator1="LRAddBlog#BLOG_CATEGORIZATION_DROPDOWN" />
+			<for list="tag1,tag2,tag3,tag4" param="tagNumber">
```

Comment:
This `<for>` loop statement should be one indent <-- because it is not nested with the previous function